environments/icds-new/public.yml

```diff
@@ -404,6 +404,7 @@ localsettings:
   SMS_GATEWAY_PARAMS:
   SMS_GATEWAY_URL: 
   SMS_QUEUE_ENABLED: True
+  SUMOLOGIC_URL: "{{ localsettings_private.SUMOLOGIC_URL }}"
```

Comment:
Also needs to be added into the localsettings file.

Reply:
I added it to `settings.py` and the vault file... 

https://github.com/dimagi/commcare-hq/pull/19597/files#diff-bdf3ecebd8379ca98cc89e545fc90899

Where else do I need to add it?

Reply:
In `roles/commcarehq/templates/localsettings.py.yml` (I think that path is right)

Reply:
I see, thanks!